cardano-cli/src/Cardano/CLI/EraBased/Commands/Governance/Committee.hs

```diff
@@ -53,7 +53,7 @@ data GovernanceCommitteeCreateHotKeyAuthorizationCertificateCmdArgs era =
   GovernanceCommitteeCreateHotKeyAuthorizationCertificateCmdArgs
     { eon               :: !(ConwayEraOnwards era)
     , vkeyColdKeySource :: !(VerificationKeySource CommitteeColdKey)
-    , vkeyHotKeySource  :: !(VerificationKeyOrHashOrFileOrScript CommitteeHotKey)
+    , vkeyHotKeySource  :: !(VerificationKeySource CommitteeHotKey)
```

Comment:
I'm wondering if we could replace all key sources with `VerificationKeySource`. But that's outside of the scope of this PR.

Reply:
I wondered the same and took a first look, but at a first glance there are no places that could reuse this `VerificationKeySource` type. But I will have another more systematic look later.